### PR TITLE
base image support

### PR DESCRIPTION
**Pull request by recallsong**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
base image support.
remove duplicated .gitignore content.
update go.mod.

#### Special notes for your reviewer:
```
```

#### Specified Reivewers:
/assgin @liuhaoyang 

#### Does this PR introduce a user-facing change?
```release-note
```

#### Additional documentation usage docs, etc.:
```docs

```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,26 +11,6 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Dependency directories (remove the comment below to include it)
```
Comment (sfwn) [inline .gitignore:14]: why delete these lines?

these files should be gitignore.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (sfwn) [reply .gitignore:14]: ok i see it's duplicated

**Review by liuhaoyang — APPROVED** [review]
/lgtm

**Review by liuhaoyang — APPROVED** [review]
/lgtm

**Pull request merged**